doc/classes/Array.xml

```diff
@@ -396,6 +397,21 @@
 				[b]Note:[/b] For performance reasons, the search is affected by [param what]'s [enum Variant.Type]. For example, [code]7[/code] ([int]) and [code]7.0[/code] ([float]) are not considered equal for this method.
 			</description>
 		</method>
+		<method name="find_custom" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="method" type="Callable" />
+			<param index="1" name="from" type="int" default="0" />
+			<description>
+				Returns the index of the [b]first[/b] value of the array that causes [param method] to return [code]true[/code], or [code]-1[/code] if there are none. The search's start can be specified with [param from], continuing to the end of the array.
+				[param method] is a callable that takes an element of the array, and returns a bool.
```

Comment:
```suggestion
				[param method] is a callable that takes an element of the array, and returns a [bool].
```

Reply:
I actually wanted to copy exactly how it is written in `any` for consistency, at least for the moment
```suggestion
				The [param method] should take one [Variant] parameter (the current array element) and return a [bool].
```

Reply:
I reiterate I would've preferred the above for consistency, but I will sleep soundly at night if the current description stays.